### DISCUSSION TITLE
ddtrace-run doens't send metric to datadog APM if max-tasks-per-child=1

### DISCUSSION THREAD

**Discussion by francisco-pjbank** [celery/celery#7988] [Q&A]
We have a container in AWS fargate and whenever celery is run with the following command `ddtrace-run celery -A app.worker.celery_worker worker --loglevel=debug --prefetch-multiplier 1 -O fair -c 6 --max-tasks-per-child 1` datadog APM doesn't show  any metrics related with task being executed.

Datadog agent is configured to run in same container.

If max-tasks-per-child is not used, then metrics are sent to datadog APM witout problem.

Anyone has seen this problem? Any possible solution?

**Comment by deep-saxena** [comment]
Hi @francisco-pjbank - I am also trying to send the celery logs to APM. As you mentioned it only works when we remove the —max-tasks-per-child argument from the command.

I wanted to ask if you were able to resolve this issue (i.e. keep the max-tasks-per-child)  or found an alternative to this. We are require to use datadog APM only. 

Thanks
